### PR TITLE
Rename embedded-bdf

### PR DESCRIPTION
**Pull request by rfuest**
Renames `embedded-bdf` to `eg-bdf` and adds some metadata.

### CODE REVIEW

File: bdf-parser/Cargo.toml [hunk 1]
```diff
@@ -1,23 +1,13 @@
 [package]
 name = "bdf-parser"
-description = "Parse BDF files"
+description = "BDF font parser"
 version = "0.1.0"
 edition = "2018"
-authors = ["James Waples <jamwaffles@gmail.com>"]
-repository = "https://github.com/jamwaffles/embedded-fonts"
-documentation = "https://jamwaffles.github.io/embedded-fonts"
+authors = ["James Waples <jamwaffles@gmail.com>", "Ralf Fuest <mail@rfuest.de>"]
```
Comment (rfuest) [inline bdf-parser/Cargo.toml:6]: @jamwaffles Is this OK for you? I'm not sure if I can publish a crate when I'm not an author. I'll add ownership for the core GitHub team when I've published the crate.

Comment (jamwaffles) [reply bdf-parser/Cargo.toml:6]: That's fine, although my email changed a while back :). Might as well update it while we're here.

```suggestion
authors = ["James Waples <james@wapl.es>", "Ralf Fuest <mail@rfuest.de>"]
```

**Review by jamwaffles — CHANGES_REQUESTED** [review]
Small change on my email, otherwise lgtm

**Comment by jamwaffles** [comment]
I just changed the repo name to `bdf` by the way

**Comment by rfuest** [comment]
> I just changed the repo name to `bdf` by the way

Thanks! I've also applied the change to the email address.

**Review by jamwaffles — APPROVED** [review]
I know the readme is garbage at the moment anyway, but it may be a good idea to add the license section from e-g to it:

```markdown
## License

Licensed under either of

- Apache License, Version 2.0 ([LICENSE-APACHE](LICENSE-APACHE) or http://www.apache.org/licenses/LICENSE-2.0)
- MIT license ([LICENSE-MIT](LICENSE-MIT) or http://opensource.org/licenses/MIT)

at your option.
```

but I won't stop the merge on its absence.

**Comment by rfuest** [comment]
Good idea and it doubles the amout information contained in the README :wink:.

**Pull request merged**